src/document.h

```diff
@@ -1302,7 +1331,9 @@ struct Document {
                 }
                 return nullptr;
 
+            case A_PASTECT:

```

Comment:
why is this a new separate operation?

Reply:
Yes, but why do we have this new operation at all? If not frequently needed it just clutters the menu/code, and there's other ways to work around it, like use the existing contiguous copy.

Reply:
Thank you for your reply. I use it a lot when I want to copy originally multi-line textes to one cell. Otherwise I have to copy them line by line to the clipboard and paste them from the clipboard. This would save some time.